substrate/frame/revive/src/exec.rs

```diff
@@ -1957,6 +1963,7 @@ where
 						self.exec_config,
 					)
 				};
+				log::debug!(target: crate::LOG_TARGET, "result of calling {dest_addr:?} pre-compile: {:?}", result);
```

Comment:
Please remove. We already have logging in the `run` function when each frame returned.